Logger/FileLogger.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string outPath;
+
+        public FileLogger(string outPath)
+        {
+            this.outPath = outPath;
+            if (!File.Exists(outPath))
+            {
+                File.Create(outPath).Close();
+            }
+        }
+
+        public override void Log(LogLevel logLevel, string s)
```

Comment:
When overriding methods it is best to keep the parameters named the same.
```suggestion
        public override void Log(LogLevel logLevel, string message)
```